### ISSUE TITLE
About combining the ViT-B/16 image encoder and the ViT-L/14 text encoder

### ISSUE THREAD

**Issue by 1d1x1w** [zhengli97/PromptKD#32]
Hello author, I am very interested in your work on PromptKD, and I have successfully reproduced it. However, when I tried to combine the ViT-B/16 image encoder and the ViT-L/14 text encoder, I found that the accuracy of the logits during the final testing is very low, only 1%-2%. Both encoders were not trained and only used a dimension alignment for adaptation. Could you please let me know what might be causing this issue?

**Comment by zhengli97** [comment]
What is your training setting?

**Comment by zhengli97** [comment]
Someone on Zhihu forum asked the same question. I guess that person should be you. I have already answered you, so I will close this issue here. 

The screenshot is as follows.

![image](https://github.com/user-attachments/assets/1b70266d-1939-4c8e-beea-215c49cb007a)
